docker/rabbiitmq/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM rabbitmq:3.8.3
+ENV RABBITMQ_DEFAULT_USER admin
```

Comment:
both user and password should be obtained from the "parent" environment variable (have a look at file storage service db connection example)